### DISCUSSION TITLE
Elspotpris-graf eller -tabel på Dakboard

### DISCUSSION THREAD

**Discussion by jacster0303** [rndfm/elspotpris#79] [General]
Er der nogen der med succes har implementeret Elspotpris på www.dakboard.com dashboard?
Så hører jeg meget gerne om det :-)

**Comment by rndfm** [comment]
Problemet med integrationer er pt. at udregningen laves på klienten(i browseren).

Så eneste mulighed pt. er en iframe med websiden i.

Der arbejdes på at flytte beregningen til serversiden således at der kan laves api'er.

**Reply by jacster0303** [reply]
Hvad med den aktuelle pris som RSS feed?
